### PR TITLE
[DSD-2906] Update application-default.properties

### PR DESCRIPTION
**Pull request by VSIVAKALYAN**

### CODE REVIEW

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
Need approval

**Comment by VSIVAKALYAN** [comment]
@vishwa-vyom please approve the Code Change which will resolve dsl sanity Failures on dev1 env. @syedsalman3753 need approval to merge PR.

File: application-default.properties [hunk 1]
```diff
@@ -372,7 +372,7 @@ object.store.swift.username=test
 object.store.swift.password=test
 object.store.swift.url=http://localhost:8080
 
-packet.manager.account.name=packet-manager
+packet.manager.account.name=${s3.pretext.value:}packet-manager
```
Comment (vishwa-vyom) [inline application-default.properties]: why do we need this, when object.store.s3.bucket-name-prefix is already configured ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: application-default.properties (hunk 1 above)
Comment (VSIVAKALYAN) [reply application-default.properties]: @sowmya695 please say why we are changing the code in this PR.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**